charts/backstage/backstage-rhtap-values.yaml

```diff
@@ -136,7 +136,10 @@ upstream:
       - name: NODE_OPTIONS
         value: --no-node-snapshot
       - name: BACKEND_SECRET
-        value: "{{ '{{ randAlphaNum 24 | b64enc }}' }}"
+        valueFrom:
```

Comment:
Referencing the Secret like this makes it easier for Orchestrator workflows to be configured. We can instruct people to check `backstage-redhat-developer-hub-auth` for the value required to communicate with Backstage. Also, easier for automation to grab it :)